fabric-command-api-v2/src/client/java/net/fabricmc/fabric/api/client/command/v2/ClientCommandsRegistrationCallback.java

```diff
@@ -28,20 +28,20 @@
  *
  * <p>To register some commands, you would register an event listener and implement the callback.
  *
- * <p>See {@link ClientCommandManager} for more details and an example.
+ * <p>See {@link ClientCommands} for more details and an example.
  */
-public interface ClientCommandRegistrationCallback {
-	Event<ClientCommandRegistrationCallback> EVENT = EventFactory.createArrayBacked(ClientCommandRegistrationCallback.class, (callbacks) -> (dispatcher, registryAccess) -> {
-		for (ClientCommandRegistrationCallback callback : callbacks) {
-			callback.register(dispatcher, registryAccess);
+public interface ClientCommandsRegistrationCallback {
```

Comment:
This rename is nonsensical and unnecessary considering there was no Yarn name for this concept to begin with.